cmd/nvidia-dra-controller/imex.go

```diff
@@ -279,7 +296,7 @@ func generateImexChannelPool(imexDomain string, numChannels int) resourceslice.P
 							Key:      ImexDomainLabel,
 							Operator: v1.NodeSelectorOpIn,
 							Values: []string{
-								imexDomain,
+								strings.Join([]string{imexDomain, strconv.Itoa(cliqueid)}, "."),
```

Comment:
Is there any reason not to continue passing in the true imex domain (i.el the one wth the clique ID already on it). Then no changes would be needed here.